spec/requests/bids_spec.rb

```diff
@@ -364,23 +372,6 @@
           end.to_not change { auction.bids.count }
         end
       end
-
-      context 'and the bid amount contains cents' do
-        let(:bid_amount) { 1.99 }
```

Comment:
the reason I deleted this test is that rails automatically rounds down the bid amount to the lower integer if you pass a float -- in that case, the person would see the rounded down float on the confirmation page, but not an error. is that a problem?